exe/bucky

```diff
@@ -135,6 +135,7 @@ def setup_test_cond(test_cond)
   %i[suite_name case label xlabel device].each do |k|
     test_cond[k] = test_cond[k].split(',') unless test_cond[k].nil?
   end
+  test_cond[:base_fqdn] = ENV['BUCKY_FQDN']
```

Comment:
Do we decide the name of the environment variable? 
Maybe we can just use `BASE_FQDN`, instead of `BUCKY_FQDN`?

Reply:
And maybe update the example which use the similar ENV.
https://github.com/search?q=repo%3Alifull-dev%2Fbucky-core%20ENV%5B%27FQDN%27%5D&type=code

Reply:
If BUCKY_FQDN is undefined it will be `nil`. Is it no problem?
https://github.com/lifull-dev/bucky-management/pull/54/files#diff-9ff5e741fff030a6de722256668538303671dacc0de45f54475855612756444bR3

`test_cond[:base_fqdn] = ENV['BUCKY_FQDN'] ||= ""`is better?